src/index.ts

```diff
@@ -72,6 +72,18 @@ export = (app: Probot) => {
     };
     await k8sApi.createNamespacedSecret(namespace,secret);
   });
+
+  // Respond to the GitHub app installation deleted
+  app.on("installation.deleted", async (context) =>{
+
+    // ? user story: delete repo or just the secret ?
+    // ? do we have to take care of repo
+    // TODO: check if secret exists and catch errors
```

Comment:
Can you please cleanup the TODO if we're not about to implement it? :slightly_smiling_face:

Reply:
done